lib/Controller/ContactsController.php

```diff
@@ -56,7 +56,9 @@ public function __construct(IRequest $request,
 	 * @param string $uuid
 	 */
 	public function direct(string $contact): RedirectResponse {
-		$url = $this->urlGenerator->getAbsoluteURL('/apps/contacts/' . $this->l10n->t('All contacts') . '/' . $contact);
+		$url = $this->urlGenerator->getAbsoluteURL(
+			$this->urlGenerator->linkToRoute('contacts.page.index') . $this->l10n->t('All contacts') . '/' . $contact
+		);
```

Comment:
No please, we have a dedicated route for this exact purpose.
So I can change it from contacts. I don't want to be server-dependant.

This looks more like a hack than a fix

Reply:
Ah, Am I reading from right to left today??
I swear I was convinced you went from route to absolute only! :see_no_evil: :roll_eyes: 

LGTM